nessie/src/main/java/org/apache/iceberg/nessie/NessieCatalog.java

```diff
@@ -281,46 +287,78 @@ public void renameTable(TableIdentifier from, TableIdentifier toOriginal) {
     // behavior. So better be safe than sorry.
   }
 
-  /**
-   * creating namespaces in nessie is implicit, therefore this is a no-op. Metadata is ignored.
-   *
-   * @param namespace a multi-part namespace
-   * @param metadata  a string Map of properties for the given namespace
-   */
   @Override
   public void createNamespace(Namespace namespace, Map<String, String> metadata) {
+    try {
```

Comment:
I notice metadata is ignored but the javadoc is removed, can you add it back.

Reply:
I have a follow-up PR for that part of the code and will add that note there if that's ok?